EIPS/eip-4974.md

```diff
@@ -0,0 +1,187 @@
+
+### EXP Word Choice
+EXP, or experience points, are common parlance in the video game industry and generally known among modern internet users. Allocated EXP typically confers to strength and accumulates as one progresses in a game. This serves as a fair analogy to what we aim to achieve with ERC-4974 by encouraging members of a community to have more strength in that community the more they contribute.
+
+*Alternatives Considered:  Soulbound Tokens, Soulbounds, Fungible Soulbound Tokens, Non-tradable Fungible Tokens, Non-transferrable Fungible Tokens, Karma Points, Reputation Tokens*
+
+### Participants Word Choice
+Participants have agency over their *participation* in an activity, but not over the *outcomes*. Parties to ERC-4974 contracts are not owners in the same sense as owners of ERC-20 or ERC-721 tokens. The EXP sits in their wallets, but those wallets do not directly control any use of the EXP. 
+
+*Alternatives Considered:  members, parties, contributors, players, entrants*
+
+### ERC-165 Interface
+We chose Standard Interface Detection (ERC-165) to expose the interfaces that an ERC-4974 smart contract supports.
+
+### Metadata Choices
+We have required `name` and `description` functions in the metadata extension. Name common among major token standards (namely, ERC-20 and ERC-721). We eschewed `symbol` as we do not wish them to be listed on any tickers that might tempt operators to engage in financial activities with these assets. We included a `description` function that may be helpful for games or other applications with multiple ERC-4974 tokens.
+
+We remind implementation authors that the empty string is a valid response to `name` and `description` if you protest to the usage of this mechanism. We also remind everyone that any smart contract can use the same name and symbol as your contract. How a client may determine which ERC-4974 smart contracts are well-known (canonical) is outside the scope of this standard.
+
+### Privacy
+Users identified in the motivation section have a strong need to identify how much EXP an address holds. Since EXP contracts are opt-in, we hope users will be proud of their accumulated recognition and not wish to keep it secret. Without metadata associated to individual tokens or wallets, the privacy risks of this standard are limited.
+
+## Backwards Compatibility
+We have adopted `Transfer`, `transfer`, `balanceOf`, `totalSupply`, and `name` semantics from the ERC-20 and ERC-721 specifications. An implementation may also include a function `decimals` that returns `uint8(0)` if its goal is to be more compatible with ERC-20 while supporting this standard.
+
+## Reference Implementation
+
+A reference implementation of this standard can be found at [../assets/EIP-4974/](../assets/EIP-4974/).
```

Comment:
It appears that CI wants links to specific files, not a folder.  Since there are just 3 files, easiest is to just link to all 3 here.
```
- ./_site/EIPS/eip-4974.html
  *  internally linking to ../assets/EIP-4974/, which does not exist (line 362)
     <a href="../assets/EIP-4974/">../assets/EIP-4974/</a>
  *  internally linking to ./erc-20, which does not exist (line 146)
     <a href="./erc-20">ERC-20</a>
  *  internally linking to ./erc-721, which does not exist (line 146)
     <a href="./erc-721">ERC-721</a>
```

Reply:
Thanks for checking this out. I'll update with just a single link to the main contract file. Other necessary files are referenced in that one.

Reply:
It's looking like any links to other new files within this PR is causing a problem. I'll try again with just a reference to the assets folder for now.

```
- ./_site/EIPS/eip-4974.html
  *  internally linking to ../assets/EIP-4974/ERC4974.sol, which does not exist (line 364)
     <a href="../assets/EIP-4974/ERC4974.sol">../assets/EIP-4974/ERC4974.sol</a>
htmlproofer 3.15.3 | Error:  HTML-Proofer found 1 failure!
The command "bash -ex .travis-ci.sh" exited with 1.
```